### DISCUSSION TITLE
The 'exit' type of segment is neither in documentation or scheme

### DISCUSSION THREAD

**Discussion by fireton** [JanDeDobbeleer/oh-my-posh#4505] [General]
I've exported the default theme to explore and VS Code shows an error on the 'exit' type segment saying it is not a valid value.

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/662637/e6ceb45d-e5a2-47ad-ac3a-2ee993c97296)

Also, there is no mention of this segment type in the documentation.

**Comment by JanDeDobbeleer** [comment]
@fireton it's renamed to status. I'll change the default config to also reflect that! Good catch.

**Reply by JanDeDobbeleer** [reply]
https://github.com/JanDeDobbeleer/oh-my-posh/pull/4507

**Comment by InuzDev** [comment]
Hey, I had the same issue, I don't know if you found a solution, but here is my code.
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/165960440/f323d663-359c-44f8-ba81-a01338f9de37)
